packages/expo/src/Notifications/Notifications.ts

```diff
@@ -418,16 +418,10 @@ export default {
   },
 
   async getBadgeNumberAsync(): Promise<number> {
-    if (!ExponentNotifications.getBadgeNumberAsync) {
-      return 0;
-    }
     return ExponentNotifications.getBadgeNumberAsync();
   },
 
   async setBadgeNumberAsync(number: number): Promise<void> {
-    if (!ExponentNotifications.setBadgeNumberAsync) {
```

Comment:
let's leave these in for supporting other platforms.

Reply:
Okay, it's restored.